modules/network-observability-updating-migrating.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-updating-migrating_{context}"]
+= Migrating removed stored versions of the FlowCollector CRD
+
+Network Observability Operator version 1.6 removes the old and deprecated `v1alpha1` version of the `FlowCollector` API. If you previously installed this version on your cluster, it might still be referenced in the `storedVersion` of the `FlowCollector` CRD, even if it is removed from the etcd store, which blocks the upgrade process. These references need to be manually removed.
+
+There are two options to remove stored versions:
+
+. Use the Storage Version Migrator Operator.
+. Reinstall the Network Observability Operator, which ensures that the installation is in a clean state. 
+
+.Prerequisites
+* You have an older version of the Operator installed, and you want to prepare your cluster to install the latest version of the Operator. Or you have attempted to install the Network Observability Operator 1.6 and run into the error: `Failed risk of data loss updating "flowcollectors.flows.netobserv.io": new CRD removes version v1alpha1 that is listed as a stored version on the existing CRD`.
+
+.Procedure
```

Comment:
My main takeaway here is that this is now essentially two distinct procedures to achieve the same outcome. If possible, it would be good to make them each a small proc rather than embedding two mini-procs in one list of steps (and having to refer to steps explicitly in the text).

Reply:
Since this is really a workaround to a bug, and its planned to be removed in the next release, I wanted it to be brief, and I felt like creating two procedure modules would make this sorta unnecessarily large.

Reply:
But I totally agree, I think for any more normal and long-standing content, I wouldn't necessarily want to present it like this. Two separate procedures would be the ideal